tests/integration/ceph_bucket_notification_test.go

```diff
@@ -122,38 +110,98 @@ func (s *ObjectSuite) TestBucketNotificationsInOrder() {
 	})
 
 	t.Run("check CephBucketNotification created for bucket", func(t *testing.T) {
-		var s3client *rgw.S3Agent
-		s3endpoint, _ := helper.ObjectClient.GetEndPointUrl(namespace, storeName)
-		s3AccessKey, _ := helper.BucketClient.GetAccessKey(obcName)
-		s3SecretKey, _ := helper.BucketClient.GetSecretKey(obcName)
-		if objectStore.Spec.IsTLSEnabled() {
-			s3client, err = rgw.NewInsecureS3Agent(s3AccessKey, s3SecretKey, s3endpoint, rgwcontext.ZoneGroup, true)
-		} else {
-			s3client, err = rgw.NewS3Agent(s3AccessKey, s3SecretKey, s3endpoint, rgwcontext.ZoneGroup, true, nil)
-		}
+		notificationPresent := utils.Retry(12, 2*time.Second, "notification is created for bucket", func() bool {
+			return helper.BucketClient.CheckBucketNotifications(namespace, storeName, obcName, bucketname, notificationName, s3endpoint)
+		})
+		assert.True(t, notificationPresent)
+		logger.Info("CephBucketNotification created successfully on bucket")
+	})
+
+	t.Run("adding new label to the ObjectBucketClaim", func(t *testing.T) {
+		obc, err := k8sh.BucketClientset.ObjectbucketV1alpha1().ObjectBucketClaims(obcNamespace).Get(ctx, obcName, metav1.GetOptions{})
+		assert.Nil(t, err)
+		obc.Labels["test-label"] = "test-value"
```

Comment:
this does not seem to be a bucket notification label.
we should expect no change here

Reply:
i see this is covered in another test